CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Changes in Version 0.2.3 (2020-xx-xx)
+=====================================
+- Fixed setup.py not finding f2py on some OSX installs
+
 Changes in Version 0.2.2 (2020-xx-xx)
 =====================================
 - Document end of support for Python 2
```

Comment:
Just stick it in 0.2.2...if we merge this after the 0.2.2 release, we can fix it then, but I'll still be doing 0.2.2 off master (the RC won't be the final.)